lang/cs/Org.Apache.REEF.IO/FileSystem/Hadoop/HadoopFileSystem.cs

```diff
@@ -152,6 +152,17 @@ public void CreateDirectory(Uri directoryUri)
             _commandRunner.Run("dfs -mkdir " + directoryUri);
         }
 
+        public bool IsDirectory(Uri uri)
+        {
+            // TODO[JIRA REEF - 2039]: HadoopFileSystem .IsDirectory() check needs to work on linux machines.
+            var stdOut = _commandRunner.Run("dfs -test -d " + uri + "& call echo %^errorlevel%").StdOut;
+            if (stdOut.Any())
```

Comment:
minor nit pick, this if statement could be reduced to return stdOut.Any() && stdOut.First() == '0';

Reply:
fixed